src/append/syslog.rs

```diff
@@ -107,8 +107,8 @@ fn log_level_to_otel_severity(level: log::Level) -> fasyslog::Severity {
     match level {
         log::Level::Error => fasyslog::Severity::ERROR,
         log::Level::Warn => fasyslog::Severity::WARNING,
-        log::Level::Info => fasyslog::Severity::INFORMATIONAL,
-        log::Level::Debug => fasyslog::Severity::DEBUG,
```

Comment:
Common mapping.

May or may not we can later add a custom mapping config, but only until there is real-world demand.